File: Selenium/Activities/Activity19.java
package activities;
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
        import org.openqa.selenium.WebDriver;
        import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
        import org.openqa.selenium.firefox.FirefoxOptions;

public class Activity19 {
    public static void main(String[] args) {
        WebDriverManager.chromedriver().setup();
        ChromeDriver driver= new ChromeDriver();

        // Open the webpage
        driver.get("https://v1.training-support.net/selenium/javascript-alerts");

        // Get the title of the page and print it
        String pageTitle = driver.getTitle();
        System.out.println("Page Title: " + pageTitle);

        // Find the button to open a CONFIRM alert and click it
        WebElement confirmButton = driver.findElement(By.id("confirm"));
        confirmButton.click();

        // Switch the focus to the alert
        org.openqa.selenium.Alert alert = driver.switchTo().alert();

        // Get the text in the alert and print it
        String alertText = alert.getText();
        System.out.println("Alert Text: " + alertText);

        // Close the alert with OK
        alert.accept();

        // Reopen the CONFIRM alert
        confirmButton.click();

        // Switch the focus to the alert again
        alert = driver.switchTo().alert();

        // Close the alert with Cancel
        alert.dismiss();

        // Close the browser
        driver.quit();
    }
}

File: Selenium/Activities/Activity19.py
from  selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.chrome.service import Service as ChromeService
from selenium.webdriver.support.select import Select
from webdriver_manager.chrome import ChromeDriverManager
driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()))

# Open the webpage
driver.get("https://v1.training-support.net/selenium/javascript-alerts")

# Get the title of the page and print it
pageTitle = driver.title
print("Page Title:", pageTitle)


# Find the confirm button and click it
driver.find_element(By.ID, "confirm").click()
# Switch focus to the alert
confirm_alert = driver.switch_to.alert
# Print the text in the alert
print(confirm_alert.text)
# Close the alert with either one of the methods
# with OK
confirm_alert.accept()
# with Cancel
# confirm_alert.dismiss()